src/makeCalculator.js

```diff
@@ -38,6 +38,34 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, number) {
+      calculator.result = callback(number);
+
+      return calculator;
+    },
+    reset(number) {
+      calculator.result = 0;
+
+      return calculator;
+    },
+    add(number) {
+      return calculator.result + number;
+    },
+    subtract(number) {
+      return calculator.result - number;
+    },
+    divide(number) {
+      return calculator.result / number;
+    },
+    multiply(number) {
+      return calculator.result * number;
+    },
```

Comment:
let's use `this.  ...` in methods instead of `calculator`

(and don't forget to bind your callback)